scripts/generate_packages_to_prefetch.py

```diff
@@ -183,16 +192,20 @@ def filter_hashes(hashes, package_line, supported_tags):
                 # Find the best matching tag based on priority
                 common_tags = wheel_tags & supported_tags
                 if common_tags:
-                    best_tag = select_best_tag(common_tags)
-                    print(f"Best tag selected for {file_info['filename']}: {best_tag}")
-                    return hash_value
+                    # register hash
+                    filtered_hashes.append(hash_value)
+
+                    # TODO: this is no longer applicable in multi-arch environments!!!
+                    # best_tag = select_best_tag(common_tags)
```

Comment:
shall we remove the definition of this function `select_best_tag`, too?
https://github.com/openshift/lightspeed-service/blob/b440e6748f097d1d720c43d4a9225aecd1319210/scripts/generate_packages_to_prefetch.py#L231

Reply:
I was thinking about it. It IS useful for one architecture scenario. And TBH I can imagine situation when we create multiple requirements files - one per architecture (and frankly, it would be the best solution for the future, need to talk about it on arch call)